packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.spec.tsx

```diff
@@ -347,6 +347,24 @@ describe("horizontal overflow", (): void => {
             0.0004
         );
     });
+    test("should ease the animation correctly when at animation start time", () => {
```

Comment:
I know it's rife through this file, but I prefer to leave out the words "correct" and "correctly" rather "should do x" would suffice since all actions should be performed correctly. In this case I think it might be better to re-title the test "should use an ease animation at the animation start time".